api/game/gameHelpers/faceoffRetrieval.js

```diff
@@ -55,6 +55,7 @@ const addSubmissionsToFaceoffs = async (conn, faceoffs) => {
     const s1 = await dbOps.getSubByID(conn, faceoffs[f].SubmissionID1);
     const s2 = await dbOps.getSubByID(conn, faceoffs[f].SubmissionID2);
     // Add formatted submissions to the faceoffs object reference passed into the function
+    //ERRLOG: submission1 instead of submissionID1(??)
```

Comment:
Thanks for figuring out where things were breaking. We have a much better idea of where to go from here.

Reply:
Yes, thank you.